.github/ISSUE_TEMPLATE/bug_report.ja.yml

```diff
@@ -21,8 +22,8 @@ body:
       required: true
   - type: input
     attributes:
-      label: "機能はすでに存在しますか？リンクを貼ってください。"
-      description: "例: https://yamada-components.app/colors/ffffff"
+      label: "コンポーネントはすでに存在しますか？リンクを貼ってください。"
+      description: "例: https://yamada-components.app/layouts/headers/simple-header"
   - type: dropdown
```

Comment:
@hirotomoyamada 
The link https://yamada-components.app/layouts/headers/simple-header appears to be invalid. 
No Problem?

Reply:
@odendayoko 

This is not a problem.
It is only a sample.